### ISSUE TITLE
ScreenPop Visualforce Page

### ISSUE THREAD

**Issue by aflint-stratusg** [developerforce/open-cti-demo-adapter#6]
Hi, I'm trying to use the demo to work with Remedyforce and the Open CTI Demo for Classic worked by utilizing the Softphone settings and Screen Popping a Visualforce Page.  Is this possible with this Lightning Open CTI Demo Adapter?

**Comment by dlouvton** [comment]
Yes, you can use searchAndScreenPop instead of runApex to search for a number and then screen pop to the caller account.

Here is a code sample you can use instead of the current code in callInitiatedPanelHelper.js that is using runApex to get contacts: 

screenPopAndCall : function(cmp) {
        cmp.getEvent('getSettings').setParams({
            callback: function(settings) {
                sforce.opencti.screenPop({
                    type : sforce.opencti.SCREENPOP_TYPE.SOBJECT,
                    params : { recordId : cmp.get('v.recordId') },
                    callback : function(response) {
                        cmp.getEvent('editPanel').setParams({
                            label : 'Open CTI Softphone: ' + cmp.get('v.state')
                        }).fire();
                        var providerClass = settings['/reqPhoneDemoSettings/reqProvider'];
                        var account = settings['/reqPhoneDemoSettings/reqProviderAccount'];
                        var token = settings['/reqPhoneDemoSettings/reqProviderAuthToken'];
                        var fromNumber = settings['/reqPhoneDemoSettings/reqProviderCallerNumber'];
                        var toNumber = cmp.get('v.phone');
                        sforce.opencti.runApex({
                            apexClass : 'SoftphoneProviderController',
                            methodName : 'call',
                            methodParams : 'providerClass=' + providerClass + '&account=' + account + '&token='+ token + '&fromNumber=' + fromNumber + '&toNumber=' + toNumber,
                            callback : function(result) {
                                if (result.success) {
                                    sforce.opencti.screenPop({
                                        type : sforce.opencti.SCREENPOP_TYPE.SOBJECT,
                                        params : { recordId : cmp.get('v.recordId') },
                                    });
                                } else {
                                    throw new Error('Unable to make a call. Contact your admin.');
                                }
                            }
                        });
                    }
                })
             }
        }).fire();
    },
